### PR TITLE
Add support for HMIP-MOD-TM

### PR DESCRIPTION
**Pull request by SukramJ**
This PR adds support for HMIP-MOD-TM (relates to #272).
I hope you haven't started to implement.

I don't own such device, so please have a look especially at the command.
Is it implemented correct?
Are there other commands?

### CODE REVIEW

File: homematicip_demo/fake_cloud_server.py [hunk 1]
```diff
@@ -584,6 +584,24 @@ def errorCode(message: str, status_code: int) -> web.Response:
...
+        js = json.loads(request.data)
+        d = self.data["devices"][js["deviceId"]]
+        channelIndex = "{}".format(js["channelIndex"])
+        switcher = {
+            "CLOSE": "CLOSED",
+            "OPEN": "OPEN",
+            "PARTIAL_OPEN": "VENTILATION_POSITION",
+            "STOP": "POSITION_UNKNOWN",
+        }
+        d["functionalChannels"][channelIndex]["doorState"] = switcher.get(js["doorCommmand"], "Invalid month")
```
Comment (coreGreenberet) [inline homematicip_demo/fake_cloud_server.py]: I think "invalid month" is wrong here

File: tests/test_devices.py [hunk 2]
```diff
@@ -1150,3 +1150,19 @@ def test_light_sensor(fake_home: Home):
...
+
+def test_door_sensor_tm(fake_home: Home):
+    with no_ssl_verification():
+        d = fake_home.search_device_by_id("3014F0000000000000FAF9B4")
+
+        assert d.doorState == DoorState.CLOSED
+        assert d.on == False
+        assert d.processing == False
+        assert d.ventilationPositionSupported == True
+
```
Comment (coreGreenberet) [inline tests/test_devices.py]: you've added a post_hmip_device_control_sendDoorCommand procedure into the fake cloud server. Could you please implement it also in the test?
Thanks

File: homematicip/aio/device.py [hunk 3]
```diff
@@ -425,3 +425,6 @@ class AsyncAccelerationSensor(AccelerationSensor, AsyncDevice):
         return await self._connection.api_call(
             *super().set_notification_sound_type(soundType, isHighToLow, channelIndex)
         )
+
+class AsyncGarageDoorModuleTormatic(GarageDoorModuleTormatic, AsyncDevice):
+    """ HMIP-MOD-TM (Garage Door Module Tormatic) """
```
Comment (coreGreenberet) [inline homematicip/aio/device.py]: async version of send_door_command is missing

File: homematicip_demo/fake_cloud_server.py (hunk 1 above)
Comment (SukramJ) [reply homematicip_demo/fake_cloud_server.py]: fixed

File: tests/test_devices.py (hunk 2 above)
Comment (SukramJ) [reply tests/test_devices.py]: test added

File: homematicip/aio/device.py (hunk 3 above)
Comment (SukramJ) [reply homematicip/aio/device.py]: async command added

**Comment by coreGreenberet** [comment]
Thx

**Comment by SukramJ** [comment]
It some time ago since the latest release.
To you have any plans for a new one?

**Comment by coreGreenberet** [comment]
yeah i want to finish the implementation of #259 and then publish a new release. I hope I have time on the next weekend for that

**Comment by coreGreenberet** [comment]
@sukramj done

**Pull request merged**